guides/saas-retention-tracking-and-churn-with-powerbi.md

```diff
@@ -0,0 +1,102 @@
+---
+description: Sigma is a cloud-native analytics platform that uses a familiar spreadsheet interface to give business users instant access to explore and get insights from their cloud data warehouse. It requires no code or special training to explore billions of rows, augment with new data, or perform “what if” analysis on all data in real⁠-⁠time.
```

Comment:
I think this may be left over from the Sigma guide. Mind updating to refer to Power BI?